### ISSUE TITLE
site.yml: The use of 'static' has been deprecated.

### ISSUE THREAD

**Issue by leseb** [ceph/ceph-ansible#2998]
Hello,

Running a syntax check on 'site.yml' using ansible 2.6.2 produces a 
deprecation warning:

```
ceph-ansible]$ ansible-playbook site.yml --syntax-check
[DEPRECATION WARNING]: The use of 'static' has been deprecated. Use 
'import_tasks' for static inclusion,
  or 'include_tasks' for dynamic inclusion. This feature will be removed 
in a future release. Deprecation
  warnings can be disabled by setting deprecation_warnings=False in 
ansible.cfg.

playbook: site.yml
```

I did not found 'static' in the playbook itself. It may be set in one of 
the referenced roles and tasks.

The playbook respectively the files in question should be changed 
accordingly before the feature will be removed.

This messages seems not to be worthy to create a bug report or feature 
request on GitHub. Unfortunately I'm not able to solve this by myself. 
So I would appreciate if the reason for this warning could be discussed 
and fixed by the members of this list.

Best regards,
Joerg K.

**Comment by leseb** [comment]
@rishabh-d-dave whenever you have time :). Thanks.

**Comment by rishabh-d-dave** [comment]
Sure.

**Comment by rishabh-d-dave** [comment]
Fixed at - https://github.com/ceph/ceph-ansible/pull/3016.

**Comment by rishabh-d-dave** [comment]
Closing this issue since it got fixed with PR #3016
